### ISSUE TITLE
Two mobility errors found in Google Search Console for LC site

### ISSUE THREAD

**Issue by libcce** [LibraryCarpentry/librarycarpentry.github.io#39]
Content wider than screen
This report indicates pages where horizontal scrolling is necessary to see words and images on the page. This happens when pages use absolute values in CSS declarations, or use images designed to look best at a specific browser width (such as 980px). To fix this error, make sure the pages use relative width and position values for CSS elements, and make sure images can scale as well. Read more in Size Content to Viewport.

Clickable elements too close together
This report shows the URLs for sites where touch elements, such as buttons and navigational links, are so close to each other that a mobile user cannot easily tap a desired element with their finger without also tapping a neighboring element. To fix these errors, make sure to correctly size and space buttons and navigational links to be suitable for your mobile visitors. Read more in Size Tap Targets Appropriately.

**Comment by maneesha** [comment]
Assigning to @kariljordan to consider as part of the The Carpentries Equity, Inclusion, and Accessibility Roadmap

**Comment by maneesha** [comment]
Assigning this to @brynnelliott - not sure what your workflow or workload is like, so prioritize as you see fit

**Comment by maneesha** [comment]
Closing as we are now using a new theme and design.
